heat/core/arithmetics.py

```diff
@@ -577,6 +653,38 @@ def invert(a: DNDarray, out: DNDarray = None) -> DNDarray:
 """Alias for :py:func:`invert`"""
 
 
+def lcm(
+    a: DNDarray, b: DNDarray, /, out: Optional[DNDarray] = None, *, where: Optional[DNDarray] = True
+) -> DNDarray:
+    """
+    Returns the lowest common multiple of |a| and |b|
+
+    Parameters
+    ----------
+    a:   DNDarray
+         The first input array
+    b:   DNDarray
+         the second input tensor
+    out: DNDarray, optional
+        The output array. It must have a shape that the inputs broadcast to and matching split axis.
+        If not provided, a freshly allocated array is returned.
+    where: DNDarray, optional
+        Condition to broadcast over the inputs. At locations where the condition is True, the `out` array
+        will be set to the divided value. Elsewhere, the `out` array will retain its original value. If
+        an uninitialized `out` array is created via the default `out=None`, locations within it where the
+        condition is False will remain uninitialized. If distributed, the split axis (after broadcasting
+        if required) must match that of the `out` array.
+
+    Examples
+    --------
+    >>> a = ht.array([6, 12, 15])
+    >>> b = ht.array([3, 4, 5])
+    >>> ht.lcm(a,b)
+    DNDarray([ 6, 12, 15], dtype=ht.int64, device=cpu:0, split=None)
+    """
+    return _operations.__binary_op(torch.lcm, a, b, out, where)
```

Comment:
Would it make sense to add a check whether the inputs have integer datatype?

Reply:
The error message from PyTorch is clear for once, so I didn't bother to add it.
> RuntimeError: "lcm_cpu" not implemented for 'Float'